CMakeLists.txt

```diff
@@ -298,7 +298,7 @@ install (FILES ${CMAKE_SOURCE_DIR}/src/pwpolicy.conf
 
 # Schema formats.
 
-install (FILES src/schema_formats/RNC/rnc.xsl
+install (FILES src/schema_formats/rnc.xsl
```

Comment:
@kslastnikov , did you apply this change manually? If so, why?

Reply:
I did not.